src/wallet/rpcdump.cpp

```diff
@@ -363,16 +363,16 @@ UniValue importaddresses(const UniValue &params, bool fHelp)
             "to spend.\n"
             "\nArguments:\n"
             "1. \"rescan | no-rescan\" (string, optional default rescan) If \"no-rescan\", skip wallet rescan\n"
-            "1. \"address\"           (string, 0 or more) The address or hex-encoded P2SH script\n"
+            "2. \"address\"           (string, 0 or more) The address or hex-encoded P2SH script\n"
```

Comment:
Shouldn't this be more like "2..." or so?

Reply:
I looked and the only other RPC call that takes a params[] style list like this (rather than a JSON Array) is `importprivatekeys` and it also follows this convention.  If we want to change we should update both places for consistency.

I'm not sure what is a standard convention for a help parameter that is a list of all subsequent parameters in the method call.  Using the ellipse seems very subtle, though I don't really have a better idea.

I also noticed that the way the method is written, adding an address is optional if you explicitly specify the resacn|no-rescan option, it would pass all the parameter checks, then fall through (and perform a rescan if that was the option) without attempting to add any addresses.  I think that maybe this should be fixed too to require at least one address, though I do sometimes find it useful to be able to initiate a rescan without needing to import anything or restart the node.

Reply:
> I'm not sure what is a standard convention for a help parameter that is a list of all subsequent parameters in the method call. Using the ellipse seems very subtle, though I don't really have a better idea.

Fair point. A "2." feels somewhat wrong to me but then this is a very minor thing. 

> I also noticed that the way the method is written, adding an address is optional if you explicitly specify the resacn|no-rescan option, it would pass all the parameter checks, then fall through (and perform a rescan if that was the option) without attempting to add any addresses. I think that maybe this should be fixed too to require at least one address, though I do sometimes find it useful to be able to initiate a rescan without needing to import anything or restart the node.

Hmm, maybe adding another line to the docs that you can do a rescan this way would be good to have then, to make this use-case explicit? I have no strong opinion either ...

Reply:
I added an extra line to the help text for rescan without importing.  It turns out that functionality is explicitly used in this manner for the QA tests.